src/mock/MockBuilder.ts

```diff
@@ -19,7 +21,7 @@ export class MockBuilder {
             } else {
                 ctor.apply(inst, []);
             }
-            this.setDefaultVals(inst, args);
+            MockBuilder.setDefaultVals(inst, args);
```

Comment:
This line was compiling to use a "this", which does not make sense on a static method